articles/openshift/oshift-how-add-domains-proxy-whitelist.md

```diff
@@ -0,0 +1,130 @@
+---
+title: How to permit outbound access to internet hosts in deployments with government community network connectivity | UKCloud Ltd
+description: Outlines methods to securely permit access to internet hosts in v3.11 OpenShift clusters deployed with government community network connectivity.
+services: openshift
+author: Ben Bacon
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Permit outbound internet access in deployments with government community network connectivity
+toc_fullpath: How To/oshift-how-add-domains-proxy-whitelist.md
+toc_mdlink: oshift-how-add-domains-proxy-whitelist.md
+---
+
+# How to permit outbound access to internet hosts in deployments with government community network connectivity
+
+## Overview
+
+This article outlines the necessary steps to whitelist domain names within the Squid proxy. Squid is an enabled service within v3.11 OpenShift clusters deployed with connectivity to government community networks (such as: HSCN, PSN, Janet). Whitelisted domains can be accessed on the internet via the proxy, enabling you to request external resources/data on nodes that previously only had connectivity to government community networks.
+
+## Prerequisites
+
+To complete the steps in this guide, you must have access to a cluster that is running OpenShift Container Platform v3.11 (or newer) with connectivity to a government community network. Squid proxy is not currently enabled within environments that do not have access to a government community network.
+
+You must also have access to either:
+
+- the OpenShift Web console
+
+- `oc`, the OpenShift command-line client (CLI). For more information, see OpenShift's [*Get Started with the CLI*](https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html)
+
+## Destination domain whitelist
```

Comment:
There should be something in this section stating the defaults that are enabled to allow the cluster to function (this should just be registry.access.redhat.com or something I think?). Stating that this access cannot be revoked as its required for container builds etc.